aspnetcore/tutorials/web-api-help-pages-using-swagger/sample/TodoApi/Startup.cs

```diff
@@ -30,8 +30,8 @@ public void ConfigureServices(IServiceCollection services)
                 });
 
                 // Set the comments path for the Swagger JSON and UI.
-                var basePath = PlatformServices.Default.Application.ApplicationBasePath;
-                var xmlPath = Path.Combine(basePath, "TodoApi.xml"); 
+                var basePath = System.AppContext.BaseDirectory;
+                var xmlPath = System.IO.Path.Combine(basePath, "TodoApi.xml"); 
```

Comment:
Note that the `System.IO` namespace is in the `using` statements at the top.

Reply:
@guardrex I missed that, mostly because on the [documentation page](https://docs.microsoft.com/en-us/aspnet/core/tutorials/web-api-help-pages-using-swagger) in that specific area the head of the file itself isn't shown. 

I guess the best would be to show the use statements at the top somehow

Reply:
Generally, we don't show them and leave it to the reader to follow the "View sample code" link. It's just done this way to keep the length of the code as short and focused as possible to aid in keeping the overall topic as short as possible. It's the same deal with namespaces (and frequently class declarations). You'll notice that the `#region snippet_ConfigureServices` here falls under both, and that's where the code will start to display in the topic.

Reply:
Agreed that we shouldn't include the inline namespaces. It clutters the code samples, and the user can access the working sample app if needed.

I recommend the following changes:
- Add `using System;` to the top of *Startup.cs*, and remove `System.` from the following line:
`var basePath = System.AppContext.BaseDirectory;`
- Remove `System.IO.` from the following line: `var xmlPath = System.IO.Path.Combine(basePath, "TodoApi.xml");